### DISCUSSION TITLE
Alternative to @tailwindcss/vite

### DISCUSSION THREAD

**Discussion by barziahmed** [vercel/turborepo#9266] [Ideas]
### Goals

To make tailwind experience better.

### Non-goals

na

### Background

na

### Proposal

Hi, 
There is a package @tailwindcss/vite which abolish needs for postcss for vite projects for using tailwind. Can we make a similar package like to do the same?

**Comment by anthonyshew** [comment]
Hi, can you explain what this has to do with Turborepo?
